local_net/local_in_place.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+source ./utils.sh
+
+alias nobled=../build/nobled
+
+HOME1=.duke
+
+CHAINID="noble-1"
+PEERS="4f9df51e0800e79e0d45fd376c11236b99be4e12@99.79.58.157:26656,3402e50ad4d838b26f8341a956c7b4b8a3c61ee5@65.109.93.44:21556"
```

Comment:
So one of these is Polkachu, but who is the other peer?

Reply:
Ours!